roadmap/2024-2025.md

```diff
@@ -1,130 +1,252 @@
+  - LLM (Gaudi TGI)
+  - Rerank
+  - Retrieval
+  - TTS
+  - VectorDB
 
-- [ ] Components contribution
-    - [ ] More micro service components per community request
-    - [ ] AI Agent support
+- **Use Cases/Examples**
+  - ChatQnA
+  - CodeGen
+  - CodeTrans
 
-- [ ] UseCases/Examples
-    - [ ] No code solution (GenAIStudio) 
-    - [ ] OPEA Model Hub
+- **Cloud Native**
+  - OneClick OPEA on ChatQnA
+  - OneClick OPEA on CodeGen
+  - GenAI microservice connector
 
-- [ ] Cloud Native
-    - [ ] Dynamic tuning on Resource management through K8s
+- **Evaluation & Others**
+  - CICD & Validation
+  - Eval: E2E (GenAIComps & GenAIExamples), lm-eval-harness, bigcode-eval-harness
+  - RAGAS evaluation service
 
-- [ ] Evaluation & Others
-    - [ ] CICD & Validation
-    - [ ] End2End evaluation 
+### AI Models
+
+- LLM: llama2 (7b, 13b, 70b), llama3 (8b, 70b), code-llama, Llama guard
+- Embedding: BGE-base
+
+### AI Tools Integration
+
+- VectorDB: Chroma
+- Framework: Langchain
+
+### Deployment Type
+
+- On Prem,IDC (Xeon, Gaudi)
+
+## June 2024
+
+### Contribution
+
+- **Components**
+  - LLM (Xeon vLLM & Ray, Ollama)
+  - OVMS
+  - prompting
+  - user feedback management
+  - Mega Component (MI6 RAG service)
+
+- **Use Cases/Examples**
+  - DocSum
+  - SearchQnA
+
+- **Cloud Native**
+  - OneClick OPEA for 2 more examples
+  - GMC with switch support (dynamic pipelines)
+  - Helm charts/templates for custom yamls (refactoring)
+
+- **Evaluation & Others**
+  - CICD & Validation
+  - Eval: E2E (GenAIComps & GenAIExamples) Gaudi (2) and CPUs in CICD cluster
+
+### AI Models
+
+- LLM: mistral-7B, mixtral-8x7B
+- Embedding: E5-mistral-7b-instruct, all-mpnet-base-v2
+
+### AI Tools Integration
+
+- VectorDB: Pinecone, Redis
+- Framework: Llamaindex, Haystack
+
+### Deployment Type
+
+- On Prem,IDC (Xeon, Gaudi)
+
+## July 2024
+
+### Contribution
+
+- **Components**
+  - LVM (Gaudi vLLM & Ray)
+  - vectordb (svs)
+  - Gateway guardrail, Auth Z/N
+
+- **Use Cases/Examples**
+  - FAQGen
+
+- **Cloud Native**
+  - OpenShift enablement for OPEA
+  - OneClick OPEA for 3 more examples
+  - Security (Service Mesh, guardrails)
+
+- **Evaluation & Others**
+  - CICD & Validation
+  - Eval: E2E (GenAIComps & GenAIExamples)
+
+### AI Models
+
+- LLM: Phi, Gemma
+- Embedding: all-MiniLM-L6-v2, paraphrase-albert-small-v2
+
+### AI Tools Integration
+
+- VectorDB: PGVector, Qdrant
+
+### Deployment Type
+
+## Aug 2024
+
+### Contribution
+
+- **Components**
+  - Documentation
+  - Test automation script
+  - Telemetry
+
+- **Use Cases/Examples**
+  - Documentation
+  - Test automation script
+
+- **Cloud Native**
+  - Demo K8s resource management
+  - Documentation on autoscaler analysis
+
+- **Evaluation & Others**
+  - CICD & Validation
+  - Eval: E2E (GenAIComps & GenAIExamples)
+
+### AI Models
+
+- Vision: llava
+- Mixtral-8x22B
+
+### AI Tools Integration
+
+- VectorDB: Milvus
+
+### Deployment Type
+
+- Public Cloud AWS (Xeon CPU & NV GPU)
```

Comment:
We already verified on AWS in July for CPUs. Just could not snag an Nvdia GPU (A100 or H100) on AWS.